src/transaction/log_tran_table.c

```diff
@@ -1473,6 +1475,48 @@ logtb_free_tran_mvcc_info (LOG_TDES * tdes)
   curr_mvcc_info->sub_ids.clear ();
 }
 
+#if defined (SERVER_MODE)
+/*
+ * logtb_set_session_tdes - reflect the session to the transaction
+ *
+ * return: nothing
+ *
+ *   tdes(in/out): Transaction descriptor
+ */
+static void
+logtb_set_session_tdes (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
+{
+  SESSION_PARAM *param;
+  int wait_msecs;
+
+  /* isolation level */
+  param = session_get_session_parameter (thread_p, PRM_ID_LOG_ISOLATION_LEVEL);
+  if (param)
+    {
+      tdes->isolation = (TRAN_ISOLATION) param->value.i;
+    }
+
+  /* lock timeout */
+  param = session_get_session_parameter (thread_p, PRM_ID_LK_TIMEOUT);
```

Comment:
```suggestion
  param = session_get_session_parameter (thread_p, PRM_ID_LK_TIMEOUT);
```
It seems that we can just use `prm_get_integer_value ();`, which calls `session_get_session_parameter` if needed and seems more general.

Reply:
And the following conversion code can be removed because `prm_`interface seems to do the conversion internally using `prm_msec_to_sec()`. Please check it.

Reply:
If we use `prm_` interface, it'd be better to remove this wrapping function, `logtb_set_session_tdes`. See the other usage of the parameters. They just set these like the below.:
```
 549   tran_isolation = (TRAN_ISOLATION) prm_get_integer_value (PRM_ID_LOG_ISOLATION_LEVEL);
 550   tran_lock_wait_msecs = prm_get_integer_value (PRM_ID_LK_TIMEOUT_SECS);
 ```

Reply:
I understood that the `prm_` interface like `prm_get_integer_value ()` concerns the `prm_Def` which is global to the server.
`set system parameters` (in csql) is applied for each session, and in order to fix this issue, the transaction must reflect the session after the commit occurs.
So I used `session_get_session_parameter ()` to get a value of session parameter `PRM_ID_LK_TIMEOUT` and `PRM_ID_LOG_ISOLATION_LEVEL` from the specific session.

Reply:
> And the following conversion code can be removed because `prm_`interface seems to do the conversion internally using `prm_msec_to_sec()`. Please check it.

If `wait_msecs` is LK_ZERO_WAIT (0), LK_INFINITE_WAIT (-1), LK_FORCE_ZERO_WAIT (-2), it should not be changed. 
Then, even if i replace `tdes->wait_msecs = wait_msecs * 1000` with `prm_sec_to_msec ()`, the condition (in wrapping function) seems to be necessary.
Also, the `prm_sec_to_msec ()` function is declared static and cannot be called from outside. So I think it's okay to change the value as `* 1000` conversion.
What do you think about this?

Reply:
> `set system parameters` (in csql) is applied for each session, and in order to fix this issue, the transaction must reflect the session after the commit occurs.

@Yechan0815 in `prm_get_integer_value()` -> `prm_get_value()`, it calls `session_get_session_parameter()` if needed. Didn't it work for this case?

Reply:
> > And the following conversion code can be removed because `prm_`interface seems to do the conversion internally using `prm_msec_to_sec()`. Please check it.
> 
> If `wait_msecs` is LK_ZERO_WAIT (0), LK_INFINITE_WAIT (-1), LK_FORCE_ZERO_WAIT (-2), it should not be changed. Then, even if i replace `tdes->wait_msecs = wait_msecs * 1000` with `prm_sec_to_msec ()`, the condition (in wrapping function) seems to be necessary.
> 
> Also, the `prm_sec_to_msec ()` function is declared static and cannot be called from outside. So I think it's okay to change the value as `* 1000` conversion.
> 
> What do you think about this?

I didn't mean to use `prm_sec_to_msec()` or `prm_msec_to_sec()` directly, but I expected they will be called internally in `prm_` interface. However, I was wrong and the conversion seems needed in this case! Sorry for confusing you.

Reply:
> > `set system parameters` (in csql) is applied for each session, and in order to fix this issue, the transaction must reflect the session after the commit occurs.
> 
> @Yechan0815 in `prm_get_integer_value()` -> `prm_get_value()`, it calls `session_get_session_parameter()` if needed. Didn't it work for this case?

You're right! I fixed the code.
Only `SERVER_MODE` refers to the session value, in other cases `prm_Def` (and `prm_` interface) is referenced. `SA_MODE` is also resolved this way.

Reply:
Please re-request the review if you're done fxing.

Reply:
I'm sorry. I'll remember it.